applications/nrf5340_audio/src/bluetooth/le_audio_bis_gateway.c

```diff
@@ -143,17 +146,45 @@ static struct bt_audio_stream_ops stream_ops = { .sent = stream_sent_cb,
 						 .started = stream_started_cb,
 						 .stopped = stream_stopped_cb };
 
+static void public_broadcast_features_set(uint8_t *features)
+{
+	int freq = bt_codec_cfg_get_freq(&lc3_preset.codec);
+
+	if (features == NULL) {
+		LOG_ERR("No pointer to features");
+		return;
+	}
+
+	if (IS_ENABLED(CONFIG_BT_AUDIO_BROADCAST_ENCRYPTED)) {
+		*features |= 0x01;
+	}
+
+	if (freq == STANDARD_QUALITY_16KHZ || freq == STANDARD_QUALITY_24KHZ) {
```

Comment:
In my opinion it might be better to actually not have these extra defines, and just add a comment here like "16000 Hz (16 kHz) and 24000 Hz (24 kHz) are standard quality". 

Same for 48 kHz.

Reply:
I don't like having magic numbers though.

Reply:
I definitely agree that we should avoid magic numbers.
But to me these will not be magic numbers anymore since we add comment on the line above. Then we avoid having to create 3 new #define

Reply:
I still think it's better to have such things defined at the top of the file instead of in the middle of it. In case these change in a spec rework, it will be easier to change